src/partials/message.less

```
@@ -0,0 +1,48 @@
+@import "../layout/constants.less";
+
+.messageWrapper {
+  position: relative;
+  max-width: 320px;
+  margin: 0 auto 8px 16px;
+  padding: 8px 32px 8px 8px;
+
+  font-size: 16px;
+  text-align: left;
+
+  color: @black;
+  background-color: rgba(51, 105, 243, 0.09);
```

HEX тоже позволяет задействовать альфа-канал. Можно использовать его для единообразия
```suggestion
  background-color: #3369f317;
```